### PR TITLE
refactor: code and styles refactoring

### PR DESCRIPTION
**Pull request by TejasShekar**
Refer to individual commit messages of this PR for changelogs.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,31 +1,27 @@
...
-  SignUp,
-} from "./pages";
+import { Header } from "./components";
+import { HomePage, ProductListing, Cart, Wishlist, Login, SignUp } from "./pages";
 
 function App() {
+  const { pathname } = useLocation();
   return (
     <div className="App">
       <ToastContainer
```
Comment (rishbitsnbytes) [inline src/App.jsx:12]: you can create diffferent file for toast portal, also can create custom-hook from this, way cleaner!

File: src/components/Filters.jsx [hunk 2]
```diff
@@ -138,6 +131,10 @@ export const Filters = () => {
           value={price}
           onChange={priceHandler}
         />
+        <div className="flex-center flex-col">
+          Showing products under
+          <div style={{ fontWeight: "bold" }}>{price}</div>
```
Comment (rishbitsnbytes) [inline src/components/Filters.jsx:136]: if you are already using utility classes then such inline styles can be avoided and used utilities instead!

**Review by rishbitsnbytes — APPROVED** [review]
Few suggestions commented, Rest LGTM!

File: src/components/Filters.jsx (hunk 2 above)
Comment (TejasShekar) [reply src/components/Filters.jsx:136]: I do not have utility classes for font weights in my component library. So, used inline styles here.

**Review by xii-vi — APPROVED** [review]
- UI Changes look better Now.
- You can merge.

File: src/App.jsx (hunk 1 above)
Comment (TejasShekar) [reply src/App.jsx:12]: react-toastify lib handles the toasts. So, there's no need for me to have portals & hooks for the same.

**Pull request merged**